File: sqrt().py
class Solution(object):
    def mySqrt(self, x):
        """
        :type x: int
        :rtype: int
        """
        for i in range(x):
            if i*i > x:
                return i-1
            elif i*i == x:
                return i

print Solution().mySqrt(2147395599)

File: Sqrt().cpp
class Solution {
public:
    int mySqrt(int x) {
        for(int i = 0; i<x; i++) {
        	if(i*i == x)
        		return i;
        	else if(i*i > x)
        		return i-1;
        }
    }
};